packages/flutter/test/widgets/interactive_viewer_test.dart

```diff
@@ -1436,6 +1436,36 @@ void main() {
       );
     });
 
+    testWidgets('InteractiveViewer.raw set does not translate child', (WidgetTester tester) async {
```

Comment:
There is nothing to "set" here? We only use a different constructor.

```suggestion
    testWidgets('InteractiveViewer.raw constructor does not translate child', (WidgetTester tester) async {
```